queries/heex/folds.scm

```diff
@@ -0,0 +1,5 @@
+; HEEx folds similar to HTML
```

Comment:
Maybe you can include the `html` folds here then ? If you used the same node names, this should work correctly.